extensions/scarb-snforge-test-collector/src/compilation.rs

```diff
@@ -0,0 +1,68 @@
+use anyhow::Result;
+use cairo_lang_sierra::program::{ProgramArtifact, VersionedProgram};
+use camino::Utf8Path;
+use once_cell::sync::Lazy;
+use rayon::prelude::{IntoParallelRefIterator, ParallelIterator};
+use serde::Serialize;
+
+use crate::compilation::test_collector::{collect_tests, ForkConfig, RawForkConfig, TestCase};
+use crate::crate_collection::{CrateLocation, TestCompilationTarget};
+use crate::metadata::LinkedLibrary;
+
+pub mod test_collector;
+
+static BUILTINS: Lazy<Vec<&str>> = Lazy::new(|| {
```

Comment:
Is there a reason why it is not public in compiler api?

Reply:
No idea tbh, can make a contribution to the compiler tomorrow

https://github.com/starkware-libs/cairo/blob/main/crates/cairo-lang-starknet/src/plugin/consts.rs#L53C51-L53C51

Reply:
@mkaput @maciektr actually I don't think it should be here anymore, it is a leftover from times where fuzzing isn't a thing. Would still like to merge the PR to cairo repo since we use it in forge for fuzzing. Sorry for the confusion

cc @cptartur

Reply:
These are also used to filter builtin arguments from user defined ones in fuzzing. We'll still need it in forge.